include/pistache/http.h

```diff
@@ -402,6 +402,13 @@ class ResponseWriter final {
 
   std::shared_ptr<Tcp::Peer> peer() const;
 
+  // Returns total count of HTTP bytes (headers, cookies, body) written when
+  // sending the response.  Result valid AFTER ResponseWriter.send() is called.
+  ssize_t getResponseSize() const { return sent_bytes_; }
```

Comment:
I think it would be better to move definitions of `getResponseSize` and `getResponseCode` methods to cpp file.

Reply:
I'm not 100% sure that moving the function bodies out of the headers is the right thing to do, for all of pistache, but that discussion is beyond the scope if this PR.  These are nearly-trivial getters, so I'll leave them inline for now.  You can move them if you wish, but I have concerns about the performance impact of the mass migration of these methods from .h into .cc files (it prevents the compiler from optimizing them at the call site).